eng/pipeline/cleanup-acr-images-pipeline.yml

```
@@ -0,0 +1,113 @@
+# This pipeline is used to clean up old images in our Azure Container Registries.
+# It's derived from https://github.com/dotnet/docker-tools/blob/main/eng/pipelines/cleanup-acr-images.yml
+
+trigger: none
+pr: none
+
+# For info about runtime parameters, see https://github.com/microsoft/go-infra/blob/main/docs/pipeline-yml-style.md#runtime-parameters
+parameters:
+  - name: disableTSA
+    displayName: "[Debug input] Disable TSA reporting. Use to try modifications in dev branches."
+    type: boolean
+    default: false
+
+schedules:
+  - cron: "0 5 * * *"
+    displayName: Nightly build
+    branches:
+      include:
+        - microsoft/main
+    always: true
+
+variables:
+  - template: variables/pipeline.yml
+  - template: variables/common.yml
```

Common already imports pipeline:

```suggestion
  - template: variables/common.yml
```

The order *could* change something... although it shouldn't, as far as I know. 😅